cmd/metrics/data.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 Google LLC
```

Comment:
Does this copyright stay to google?

Reply:
Updated, sorry that was auto-generated at one point.